runtime/binding-grpc/src/main/antlr4/io/aklivity/zilla/runtime/binding/grpc/internal/parser/Protobuf3.g4

```diff
@@ -46,14 +46,28 @@ packageStatement
 // Option
 
 optionStatement
-  : OPTION optionName EQ constant SEMI
+  : OPTION optionName EQ optionValue SEMI
   ;
 
 optionName
   : fullIdent
   | LP fullIdent RP ( DOT fullIdent )?
   ;
 
+optionValue
+  : constant
+  | LC optionFields? RC
+  ;
+
+optionFields
+  : optionField ( optionField )*
+  ;
+
+optionField
+  : optionName COLON optionValue
+  | optionName LC optionFields? RC
```

Comment:
Perhaps we should have this since it feels like unusual syntax to omit the colon (even though we need to support that!) ?
```suggestion
  | optionName LC COLON? optionFields? RC
```